packages/extension/test/integration/background_spec.js

```diff
@@ -428,18 +428,17 @@ describe('app/background', () => {
 
```

Comment:
Most of the changes in this files were changing `google.com` to `example.com` and cleaning up a bunch of necessary returns. Otherwise, just updating any tests that counted on the previously looser domain matching.